rosbridge_server/launch/rosbridge_websocket.launch

```diff
@@ -7,7 +7,7 @@
   <arg name="authenticate" default="false" />
   
   <group if="$(arg ssl)">
-    <node name="rosbridge_websocket" pkg="rosbridge_server" type="rosbridge_websocket" output="screen">
+    <node name="rosbridge_websocket_client" pkg="rosbridge_server" type="rosbridge_websocket_client" output="screen">
```

Comment:
@Shokman Is this on purpose? This is going to break the regular websocket server!